### PR TITLE
Update color variables on DSCO Alert component

### PR DESCRIPTION
**Pull request by kelbyhawn**
Swaps out color variables from `shared/color.scss` with utility colors from `primitiveColors.css` on the Alert component. 

## Links
Jira ticket: [DESIGN2-184](https://codedotorg.atlassian.net/browse/DESIGN2-184)

## Testing story 
Tested locally in Storybook

----

## Before - using `shared/colors.scss`
Before

## After - using `color.css`
After

### CODE REVIEW

File: apps/src/componentLibrary/alert/alert.module.scss [hunk 1]
```diff
@@ -19,50 +19,50 @@ $alert-span-vertical-top-spacing: 2px;
...
   }
 }
 
 .alert-danger {
-  background-color: $light_negative_100;
+  background-color: var(--sentiment-error-10);
 
   .alertContentContainer i {
-    color: $light_negative_500;
+    color: var(--text-error-primary);
```
Comment (kelbyhawn) [inline apps/src/componentLibrary/alert/alert.module.scss:44]: Example of the new color variable in the inspector:
Screenshot 2024-11-27 at 9 52 18 AM

File: apps/src/componentLibrary/alert/alert.module.scss [hunk 2]
```diff
@@ -19,50 +19,50 @@ $alert-span-vertical-top-spacing: 2px;
...
+  background-color: var(--brand-purple-10);
 }
 
 .alert-success {
-  background-color: $light_affirmative_100;
+  background-color: var(--sentiment-success-10);
 
   .alertContentContainer i {
-    color: $light_affirmative_500;
+    color: var(--sentiment-success-50);
```
Comment (kelbyhawn) [inline apps/src/componentLibrary/alert/alert.module.scss:36]: Example of correct variable name showing in DevTools:
Screenshot 2024-11-27 at 12 06 32 PM

**Review by levadadenys — APPROVED** [review]
LGTM! Thank you for this update!

**Comment by levadadenys** [comment]
Created a PR https://github.com/code-dot-org/code-dot-org/pull/62743 updating colors documentation

**Pull request merged**